README.md

```diff
@@ -181,6 +181,30 @@ It will produce files: `en-38.png`, `en-39.png`, `en-40.png` and `en-41.png`.
 
 **This script requires `pdftoppm` and `imagemagick` utilities.**
 
+### 🎭 Comparing two pages side by side (v2)
+
+Another version, which works faster, but uses a different workflow:
+
+First, switch to the `main` branch, build the `main_en.pdf` and run:
```

Comment:
Both scripts need this file. How about storing them online with the PDFs? Upon merging to main, pictures could be created and uploaded to GitHub.

Reply:
It's easy enough to extract the image from pdf, this is done with a single command. But downloading artifacts from github is slower/more complicated - needs github API tokens, etc.

Reply:
No it's not. It would be the same as downloading PDFs from readme. On my machine, it takes ~15 s to build the PDF. Say you want to compare 3 images, that's 6 MB? I'll download it faster than in 15 s.

Reply:
Ah, you mean not an artifact, but the separate repo. Then... maybe, but redownloading the same image again and again is still waste of bandwidth

Reply:
Well, I can afford to download 6 MB from time to time 😆 Doesn't seem like a huge cost.

Reply:
I would also prefer downloading the files - full build on my machine takes 45s which is crazy long (twice as much if I also want printable).

But downloading the latest build for the given language and caching it for a while would make the workflow easier and I wouldn't have to keep track of the baseline builds. It could also have an option to download/compare regular and `printable` versions.

Reply:
This is my attempt on this topic https://github.com/Heegu-sama/Homm3BG/pull/567.

I use it for myself but if you're interested, I can make it a proper PR

Reply:
I guess this PR can be closed since the features were implemented in https://github.com/Heegu-sama/Homm3BG/pull/567?